vuln-remediation.md

```diff
@@ -0,0 +1,105 @@
+# Vulnerability Remediation
+
+This document describes the typical process for remediating security vulnerabilies in ROS 2, including those submitted through the process  described in [REP 2006, ROS 2 Vulnerability Disclosure Policy](https://www.ros.org/reps/rep-2006.html).
```

Comment:
```suggestion
This document describes the typical process for remediating security vulnerabilies in ROS 2, including those submitted through the process described in [REP 2006, ROS 2 Vulnerability Disclosure Policy](https://www.ros.org/reps/rep-2006.html).
```

Reply:
Updated in 1143387b70d57380cf6501ac1090bf1c3f1e1411